### PR TITLE
My json file is added

### PR DESCRIPTION
**Pull request by Saha-7**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1764"></a>

### CODE REVIEW

File: public/data/PritamSaha7.json
Comment (Panquesito7) [inline public/data/PritamSaha7.json:5]: It seems this is the wrong link.
```suggestion
    "avatar": "https://github.com/Saha-7.png",
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Saha-7** [comment]
Dear maintainers,
I am really sorry for the mistakes. I have changed it committed directly to
the patch1. Should I have to Pull request again from scratch?

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉 


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Comment by Saha-7** [comment]
Thanks for the approval. Can you assign me in the issue number #1728?
I am ready with my forked repo with the updated Readme.MD file that will
help the beginners to add their profile.
Waiting for your reply
Thank you

**Comment by Pradumnasaraf** [comment]
Closing this PR. As the issue link to this PR is already assigned to someone else.

**Comment by Pradumnasaraf** [comment]
Applogies

**Pull request closed without merging**
